it/mail/chrome/messenger/am-server-top.dtd

```diff
@@ -66,15 +66,15 @@
     maxMessengerStart.label will be followed by maxMessagesEnd.label with the number
    of messages between them
 -->
-<!ENTITY maxMessagesStart1.label "Ask when downloading more than">
+<!ENTITY maxMessagesStart1.label "Chiedi durante il download di più di">
 <!ENTITY maxMessagesStart1.accesskey "m">
 <!-- LOCALIZATION NOTE (maxMessagesEnd.label) : see note for maxMessagesStart.label -->
 <!ENTITY maxMessagesEnd.label "messaggi">
-<!ENTITY alwaysAuthenticate1.label "Always request authentication">
+<!ENTITY alwaysAuthenticate1.label "Richiedi sempre l'autenticazione">
 <!ENTITY alwaysAuthenticate1.accesskey "w">
-<!ENTITY abbreviate.label "Mostra nome del gruppo di discussione nel pannello della posta così:">
-<!ENTITY abbreviateOn.label "Nomi completi (Per esempio, 'netscape.public.mozilla.mail-news')">
-<!ENTITY abbreviateOff.label "Nomi abbreviati (Per esempio, 'n.p.m.mail-news')">
+<!ENTITY abbreviate.label "Mostra nome Newsgroup nel pannello E-mail come:">
+<!ENTITY abbreviateOn.label "Nomi completi (Ad esempio, 'netscape.public.mozilla.mail-news')">
```

Comment:
CORRECTIONS NEEDED. 

The sentence isn't interrupted by a full stop - a bracket is used instead - therefore "Ad" must be replaced by "ad".